src/stun.spec.ts

```diff
@@ -9,25 +9,28 @@ import { once } from 'events'
 describe('test STUN', function () {
   let servers: Socket[]
 
-  before(() => {
-    servers = Array.from({ length: 4 }).map((_) => {
-      const server = dgram.createSocket('udp4')
-      server.on('message', (msg: Buffer, rinfo: RemoteInfo) => handleStunRequest(server, msg, rinfo))
-      server.on('error', (e) => {
-        throw e
-      })
-      return server
-    })
-  })
+  before(async () => {
+    servers = await Promise.all(
+      Array.from({ length: 4 }).map(
```

Comment:
any reason for the test size of 4?

Reply:
The idea is to start a few STUN servers and test against each other. Need to checkout if 4 is a good choice. Have changed the tests multiple times